resources/js/wishlist.js

```diff
@@ -4,7 +4,7 @@ document.addEventListener('turbolinks:load', () => {
     window.app.$on('logged-in', async () => {
         let response = await axios.post(config.magento_url + '/graphql', {
             query: '{ customer { wishlists { id items_v2 { items { id product { sku id } } } } } }'
-        }, { headers: { Authorization: `Bearer ${localStorage.token}` }})
+        }, { headers: { Authorization: `Bearer ${localStorage.token}`, Store: `${ config.store_code }` }})
```

Comment:
Since it's not part of a string the variable can be used directly
```suggestion
        }, { headers: { Authorization: `Bearer ${localStorage.token}`, Store: config.store_code }})
```